app/build.gradle

```diff
@@ -162,9 +162,11 @@ dependencies {
     implementation project(":data")
     implementation project(":qrscanner")
 
+    implementation "com.google.code.findbugs:jsr305:$versions.findbugs"
```

Comment:
Why do we need this? It would be nice to have a little bit more info on that as I don't see any annotation in this PR that uses this. Also I assume we would need findbugs for that (which is deprecated).